MyCryptoHQ/MyCrypto
# Allow BIP49 dpath

Closes #2232

### Description

It's necessary for RSK platform to access accounts that are derivated from BTC segwit addresses. Those dpath start like `m/49'/` and MyCrypto only supports `m/44'/`.

### Changes

* A Regex expression that validate custom dpaths

### Steps to Test

1. Go to "Select an Address"
2. Select "Custom" from the combo
3. Write a dpath that starts with `m/49'/` and check it gets validated
4. Write a dpath that starts with `m/44'/` and check it still gets validated
